src/llamafactory/model/loader.py

```diff
@@ -104,7 +104,7 @@ def load_tokenizer(model_args: "ModelArguments") -> "TokenizerModule":
         setattr(processor, "video_fps", model_args.video_fps)
         setattr(processor, "video_maxlen", model_args.video_maxlen)
     except Exception:
-        processor = None
+        pass
```

Comment:
We need to define the variable before using it

Reply:
I will using `hasattr` to check and set in next commit.

Reply:
my bad i havn't check the traceback XD.

Reply:
Seems is dynamic [https://github.com/huggingface/transformers/blob/0a21381ba3047882ffe1b95c639aec28974b2c7e/src/transformers/models/mllama/modeling_mllama.py#L1386](Embed using conv)

Reply:
Fixed with ((config.vision_config.image_size // config.vision_config.patch_size) ** 2 + 1) * config.vision_config.max_num_tiles